### DISCUSSION TITLE
Settings not being saved

### DISCUSSION THREAD

**Discussion by Allen050329** [DavidS95/Smokeless_UMAF#11] [Q&A] [answered]
Hello, I have been helping others with using this tool, yet some have had problems saving integrated graphics settings. The two computers in question are: Asus M409DA and ThinkBook 16p Gen2, and we are unable to save the vram size if it is above ¼ of system memory for the former, while the latter's cannot be changed at all, under all modes (legacy, non-legacy and hybrid secure). Is there any way to fix that?

**Comment by DavidS95** [comment, marked as answer]
There is no way to force the settings to work, this tool simply reveals hidden bios settings it cannot force the options to work if they do not work for whatever reason. Also you would not want to increase the "vram" size typically even if u could. What apus do both of these systems have i havent seen this issue prior.

**Comment by Allen050329** [comment]
Okay, closing
